upup/pkg/fi/nodeup/command.go

```diff
@@ -241,6 +246,14 @@ func (c *NodeUpCommand) Run(out io.Writer) error {
 		loader.Builders = append(loader.Builders, &model.CalicoBuilder{NodeupModelContext: modelContext})
 	}
 
+	loader.TemplateFunctions["MapJson"] = func(m map[string]string) (string, error) { 
```

Comment:
my text/template foo is not very strong and this is what I came up with to get my template to render correctly. I'm sure there's a better approach. This function seems out of place here